README.md

```diff
@@ -35,6 +29,32 @@ toc: true
 ---
 ```
 
+There are three Liquid filters available now.
+
+1. `toc_only`  
+   Generates the TOC itself as described [below](#generated-table-of-contents-html). 
+
+2. `inject_anchors`  
```

Comment:
It's a filter, not a tag, thus, same as with `toc`. See comment below on our usage.